forge-gui/res/cardsfolder/upcoming/darling_of_the_masses.txt

```diff
@@ -0,0 +1,11 @@
+Name:Darling of the Masses
+ManaCost:2 G W
+Types:Creature Elf Citizen
+PT:2/4
+S:Mode$ Continuous | Affected$ Creature.Citizen+Other+YouCtrl | AddPower$ 1 | Description$ Other Citizens you control get +1/+0.
+T:Mode$ Attacks | ValidCard$ Card.Self | TriggerZones$ Battlefield | Execute$ TrigToken | TriggerDescription$ Whenever CARDNAME attacks, create a 1/1 green and white Citizen creature token.
+SVar:TrigToken:DB$ Token | TokenScript$ gw_1_1_citizen
+SVar:PlayMain1:TRUE
+SVar:HasAttackEffect:True
+DeckHas:Ability$Token & Type$Citizen
```

Comment:
Forge already knows this card has Type Citizen because of the Types line, but we want it to know that including other Citizen cards is a good idea here, so:
```
DeckHas:Ability$Token
DeckHints:Type$Citizen
```